vizro-core/src/vizro/models/types.py

```diff
@@ -329,9 +331,9 @@ def wrapped(*args, **kwargs):
 
 
 # Types used for selector values and options. Note the docstrings here are rendered on the API reference.
-SingleValueType = Union[StrictBool, float, str]
+SingleValueType = Union[StrictBool, float, str, Timestamp]
```

Comment:
```suggestion
SingleValueType = Union[StrictBool, float, str, date]
```

Unless I missed something? Otherwise it's inconsistent with the model field type.

Similar for `MultiValueType`.